edi_oca/migrations/14.0.1.10.0/post-migrate.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Camptocamp SA (http://www.camptocamp.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
Why not using openupgradelib?

Reply:
because AFAIS there's no need. What's the benefit in this case?

Reply:
Well, usually it is good in order to avoid the env definition, obviously not needed, but makes it all easier IMO. Obviously this is not required